bazel/linux/run_um_kunit_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+LINUXIMG=$1
+ROOTFS=$2
+HOSTFS=$(dirname $(realpath $3))
+
+set -e
+
+mkdir -p /tmp/uml
+USER=$(whoami)
+chown "${USER}.${USER}" /tmp/uml
+chmod 777 /tmp/uml
+export TMPDIR=/tmp/uml
+
+OUTFILE="${TMPDIR}/output"
+KUNIT_TAP_PARSER=/path/to/kunit_tap_parser
```

Comment:
On the same note as the script itself, how should we expose/declare this other script? I'm guessing this will need to be declared as data in the bazel rule, and its path passed as an argument to this script.

Reply:
yes, declare as data in the bazel rule, and then pass the path as env variable, or command line parameter.

I've often seen rules use ctx.write_template to generate a shell script, expanding the paths and parameters inside to avoid / reduce impact of escape hell.

Reply:
> yes, declare as data in the bazel rule, and then pass the path as env variable, or command line parameter.

It seems that exported env variables don't carry over the bazel execution environment. I will just consider it as another path parameter.

> I've often seen rules use ctx.write_template to generate a shell script, expanding the paths and parameters inside to avoid / reduce impact of escape hell.

Given the simplicity of our script I think we would just be overengineering it (+ without templating we can run the script manually as well).